Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -13,17 +13,25 @@
     <Title>AutoFixture with Auto Mocking using NSubstitute</Title>
     <Description>This extension turns AutoFixture into an Auto-Mocking Container. The mock instances are created by NSubstitute. To use it, add the AutoNSubstituteCustomization to your Fixture instance.</Description>
     <Authors>Daniel Hilgarth,AutoFixture</Authors>
+
+    <!--  Suppress warning about invalid dependency version in Castle.Core.
+          That is NSubstitute dependency and we cannot fix that somehow. -->
+    <NoWarn>$(NoWarn);NU1603</NoWarn>
   </PropertyGroup>
 
-  <ItemGroup>
-    <CodeAnalysisDictionary Include="CodeAnalysisDictionary.xml" />
+  <ItemGroup Condition=" '$(TargetFramework)'=='net45' ">
+    <PackageReference Include="NSubstitute" Version="1.5.0" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="NSubstitute" Version="1.5.0" />
+  <ItemGroup Condition=" '$(TargetFramework)'=='netstandard1.5' ">
+    <PackageReference Include="NSubstitute" Version="2.0.3" />

```

Comment:
Isn't this a breaking change?

Reply:
No, this is not a breaking change. If you install this package to .NET Framework 4.5 you will still need NSubstitute 1.5 as a dependency requirement.

For the .NET Standard project you will need NSubstitute `2.0.3`, however .NET Standard support is a _new feature_, so nothing existing was broken.